next.config.example.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
                     api: 'http://etherpad.localhost/mypads/api',
                     spacesGroupName: '/spaces', // optional, defaults to publicRuntimeConfig.name
                 },
+                templates: ['etherpad-link'],
```

Comment:
Is this the correct default? When looking at my rooms from my test instance they all have their template set to `write-link`... I think this is another thing that needs to be addressed with https://github.com/medienhaus/medienhaus-spaces/issues/30 for more consistency.

IMHO this should be just `etherpad`. And udk/spaces knows how to interpret `etherpad`. No matter how the sysadmin/developer decided to call that "app". And `etherpad` will also always be the known by the codebase, as things are called `EtherpadAuthProvider`, `/pages/etherpad/<...>`, and so on here.
This would work **without** defining a template. The template would just always be the exact same as the technical name of the "service".